src/setupWorker/start/createStart.ts

```diff
@@ -15,6 +15,7 @@ import { createResponseListener } from '../../utils/worker/createResponseListene
 const DEFAULT_START_OPTIONS: DeepRequired<StartOptions> = {
   serviceWorker: {
     url: '/mockServiceWorker.js',
+    shared: false,
```

Comment:
Can we have MSW working in a "shared" mode by default? What implications would there be in that case?

Reply:
@kettanaito I actually have question to be honest. I've been trying to understand why a mock server can't just mock requests for all clients that connect to it. Seems like it was an intentional design decision. Does anyone have any more context on this?

Reply:
The main reason we've added the `clients` object is to give developers control over the mocking on a per-client basis. By calling `worker.stop()` you can disable mocking for the current client.